hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1072,6 +1074,19 @@ message KeyInfo {
     optional bool isFile = 19;
 }
 
+message BasicKeyInfo {
+    required string volumeName = 1;
+    required string bucketName = 2;
+    required string keyName = 3;
+    required uint64 dataSize = 4;
+    required uint64 creationTime = 5;
+    required uint64 modificationTime = 6;
+    required hadoop.hdds.ReplicationType type = 7;
+    optional hadoop.hdds.ReplicationFactor factor = 8;
+    optional hadoop.hdds.ECReplicationConfig ecReplicationConfig = 9;
+    optional bool isFile = 10;
```

Comment:
qq: how is it useful for client? Isn't `keyName` enough to tell if it is key or dir? If I'm not wrong dir ends with `/`.

Reply:
Thank you @hemantk-12, this is a good idea. I have incorporated the changes to remove the field `isFile` from the `BasicKeyInfo` proto message.